api.ts

```diff
@@ -0,0 +1,73 @@
+import { MinorityGroups, BusinessType } from "./src/models";
```

Comment:
This is pretty minor, but I don't think api.ts is the most descriptive file name. We might also want to put this in the constants folder or make an api folder,